### PR TITLE
Fixed transition counts inclusive/exclusive

### LINKED ISSUE #27: No AOI -> RPM transition proportion excluding self transitions is incorrectly 1
- "No AOI -> RPM" proportion excluding self transitions is 1 for p10, p11, p21, and more. 
- This is incorrect because it would mean the pilot's gaze only moved from No AOI to the RPM AOI. The proportions should add up to 1 and there are other non-zero transition proportions from No AOI.

### PR DESCRIPTION
**Pull request by Immersor-git**
Fixes #27

### CODE REVIEW

File: src/analysis/AreaOfInterests.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import java.util.Set;
 
 public class AreaOfInterests {
-    final static String FIXATIONID_INDEX = "FPOGID";
+    final static String FIXATIONID_INDEX = "CNT"; //FPOGID
```
Comment (ashkjones) [inline src/analysis/AreaOfInterests.java]: @Immersor-git Could you explain why CNT is used over FPOGID? Not saying its wrong but I just want to understand the code better.

**Review by ashkjones — COMMENTED** [review]
image

I tried testing with p11's data (arbitrarily chosen). I summed the number of transitions (103976) and found it was higher than the number of fixations (1369) in the DGM.csv. I don't think there should be more transitions than there are fixations.

I understand this is due to multi-AOI fixations, but I think we decided to only use the FPOGX and Y of the fixation entry (not the intermediate data records) to assign the AOI. I think you want to go back to using `allFixations` and not `allGazeData`. I could be mistaken though. Maybe ask Dr. Fu how to proceed.

**Comment by Immersor-git** [comment]
> image
> I tried testing with p11's data (arbitrarily chosen). I summed the number of transitions (103976) and found it was higher than the number of fixations (1369) in the DGM.csv. I don't think there should be more transitions than there are fixations.
> 
> I understand this is due to multi-AOI fixations, but I think we decided to only use the FPOGX and Y of the fixation entry (not the intermediate data records) to assign the AOI. I think you want to go back to using `allFixations` and not `allGazeData`. I could be mistaken though. Maybe ask Dr. Fu how to proceed.

I had switched back to allGaze instead of allFixations, as it seemed that a lot of transitions weren't registered when using allFixations. Part of the reason is related to your earlier question on using CNT over FPOGID. It turned out that FPOGID didn't increase every time there was an AOI transition, so a lot of them would be missed the lines were expected to have subsequent FPOGID values. 

I've just pushed another fix for this by counting transitions that have identical or subsequent FPOGID values, and switched back to using allFixations instead of allGaze - the resulting numbers appear much more reasonable now.

image

**Comment by Immersor-git** [comment]
@ashkjones 
Pushed on final update - uses FPOGID again, checks for subsequency between FPOGIDs only, and uses allFixations instead of allGaze. This should be the final patch required.

**Pull request merged**